spark/src/main/java/org/apache/iceberg/spark/source/Writer.java

```diff
@@ -59,6 +51,14 @@
 import org.apache.spark.sql.sources.v2.writer.WriterCommitMessage;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.function.Function;
```

Comment:
Why did these imports move?

Reply:
Most likely IDE moved them. Do we have a standard style for these and a way to enforce it?